pkg/server/skydb/pq/builder/sort.go

```diff
@@ -26,7 +26,11 @@ func SortOrderBySQL(alias string, sort skydb.Sort) (string, error) {
 
 	switch sort.Expression.Type {
 	case skydb.KeyPath:
-		expr = fullQuoteIdentifier(alias, sort.Expression.Value.(string))
+		if len(sort.Expression.KeyPathComponents()) == 2 {
```

Comment:
Please handle `len(expr.KeyPathComponents()) > 2` explicitly.

Reply:
It's not used, removed.